### PR TITLE
DOCKER - Latest versions of chrome and hub

### PR DESCRIPTION
**Pull request by omarlopesino**
This PR upgrades node containers and adds needed patches to make selenium driver compatible with it.

### CODE REVIEW

File: patches/mink-361-selenium-driver-switch-to-iframe-1.patch [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+diff --git a/src/Selenium2Driver.php b/src/Selenium2Driver.php
+index 4f5cc759..b48bacc0 100755
+--- a/src/Selenium2Driver.php
++++ b/src/Selenium2Driver.php
+@@ -430,7 +430,7 @@ public function switchToIFrame($name = null)
+           ];
+         }
+         else {
+-          $element_info = NULL;
++          $element_info = null;
```
Comment (Eduardo-Morales-Alberti) [inline patches/mink-361-selenium-driver-switch-to-iframe-1.patch:53]: @mistermoper why null are lowercase in some sites and uppercase in other places like https://github.com/Metadrop/drupal-boilerplate/pull/106/files#diff-150ac77db3caab46bb2132ed3502198eb1fe25b29b9b82c3738dfec636916d47R27

**Comment by Eduardo-Morales-Alberti** [comment]
Seems right to me, did you test on any real project, and did it work? @mistermoper

**Comment by omarlopesino** [comment]
Yes I tested it in three projects and they are working fine with this versions and patches.

**Comment by jorgetutor** [comment]
looks good to be too

**Pull request merged**